tests/DistortionTests.cpp

```diff
@@ -177,3 +178,92 @@ TEST(DawnFc, testRemove) {
   EXPECT_NEAR(ux,10.0,1e-8);
   EXPECT_NEAR(uy,10.0,1e-8);
 }
+
+TEST(DawnFc, testZeroCoeffs) {
+  csm::ImageCoord imagePt(10.0, 10.0);
+
+  double ux, uy, dx, dy;
+  double desiredPrecision = 0.0000001;
+  std::vector<double> coeffs = {0};
+
+  applyDistortion(imagePt.samp, imagePt.line, dx, dy, coeffs,
+                  DistortionType::DAWNFC, desiredPrecision);
+
+  removeDistortion(dx, dy, ux, uy, coeffs,
+                  DistortionType::DAWNFC, desiredPrecision);
+
+
+  EXPECT_NEAR(dx,10.0,1e-8);
+  EXPECT_NEAR(dy,10.0,1e-8);
+  EXPECT_NEAR(ux,10.0,1e-8);
+  EXPECT_NEAR(uy,10.0,1e-8);
+}
+
+TEST(KaguyaTc, testApply) {
+  csm::ImageCoord imagePt(2.0, 2.0);
+
+  double dx, dy;
+  double desiredPrecision = 0.0000001;
+  std::vector<double> distortionCoeffs = {1, 0, 0, 0,
+                                          1, 0, 0, 0};
+
+  applyDistortion(imagePt.samp, imagePt.line, dx, dy, distortionCoeffs,
+                  DistortionType::KAGUYATC, desiredPrecision);
+
+  EXPECT_NEAR(dx, 1, 1e-8);
+  EXPECT_NEAR(dy, 1, 1e-8);
+}
+
+TEST(KaguyaTc, testRemove) {
+  csm::ImageCoord imagePt(1.0, 1.0);
+
+  double ux, uy;
+  double desiredPrecision = 0.0000001;
+  std::vector<double> distortionCoeffs = {1, 0, 0, 0,
+                                          1, 0, 0, 0};
```

Comment:
We're only checking that the first term is being handled correctly, we should test non-zero coefficients for the other terms too.

Reply:
I'm not sure how necessary it is to actually test the individual coefficients like this. The two tests bellow are applying and removing distortion with actual coefficients and all zero coefficients. Both of those seem to be covering all of our bases